client/src/App.tsx

```diff
@@ -9,3 +9,32 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
     <Router />
   </React.StrictMode>
 );
+
+interface Listitem {
+  text: string;
+}
+
+function createUL(listItems: Listitem[]): void {
```

Comment:
React has custom ways to create a component for objects within an array by calling `map` on the array that we should use, it's a lot cleaner and avoids raw JS DOM manipulation. @kravich2 's [recent PR here](https://github.com/varangill/ClubHub/pull/9/files#diff-26cee6f83578cbe23a73e0f3c317f0764a2a250bc18cc362c69436f42c804730R67) has an example of it

Reply:
Also this sort of page-specific logic shouldn't be in `App.tsx`